index.html

```diff
@@ -128,6 +128,87 @@ <h2>
 			</div>
 		</section>
 
+		<!-- Five -->
+		<section class="spotlight style1 orient-left content-align-left image-position-center onscroll-image-fade-in">
+			<div class="content">
+				<h2>
+					Presentations & Publications
+				</h2>
+				<ul class="fa-ul">
+					<li><span class="fa-li"><i class="fas fa-microphone fa-2x"></i></span>
+						<span style="font-family: 'Roboto', sans-serif;"><span
+							style="font-weight: 300;">R</span><span style="font-weight: 400;">I</span><span
+							style="font-weight: 500;">S</span><span style="font-weight: 700;">E</span></span> 
+						: A Functional Pattern-based Dialect in MLIR 
+						<br>
+						Martin Lücke
+						<br>
+						<a href="https://docs.google.com/document/d/1y_9f1AbfgcoVdJh4_aM6-BaSHvrHl8zuA5G4jv_94K8/"><i class="fab fa-2x" aria-hidden="true"></i>
+							MLIR Open Design Meeting 05/2020</a>	
+						<br>
+						<a href="http://rise-lang.org/presentations/2020/accml_RISE_in_MLIR_Bologna_2020.pdf" style="margin-right: 1em;">
+						<span class="fa-stack fa-1x">
+							<i class="fas fa-file-powerpoint fa-stack-2x" style="font-size:36px"></i>
+						</span>
+						Slides</a>
+						<a href="https://drive.google.com/open?id=18DNvegfHcEQfkKktkBhmhUecyNunfy2s" style="margin-right: 1em;">
+						<span class="fa-stack fa-1x">
+							<i class="fas fa-file-video fa-stack-2x" style="font-size:36px"></i>
+						</span>
+						Recording</a>
+					</li><br>
+
+					<li><span class="fa-li"><i class="far fa-image fa-stack-2x"></i></span>
+						<span style="font-family: 'Roboto', sans-serif;"><span
+							style="font-weight: 300;">R</span><span style="font-weight: 400;">I</span><span
+							style="font-weight: 500;">S</span><span style="font-weight: 700;">E</span></span> in MLIR - A functional pattern-based dialect 
+						<br>
+						Martin Lücke, Michel Steuwer
+						<br>
+						<a href="https://www.c4ml.org/c4ml2020"><i class="fab fa-2x" aria-hidden="true"></i>
+							2nd C4ML workshop, at CGO 2020</a>	
+						<br>
+						<a href="http://rise-lang.org/presentations/2020/accml_RISE_in_MLIR_Bologna_2020.pdf" style="margin-right: 1em;">

```

Comment:
Change the link to the poster first.

Reply:
Fixed in https://github.com/rise-lang/rise-lang.github.io/pull/4/commits/761105828a4e58d105b14475a3457ce0703896aa